frontend/src/app/modules/bim/ifc_models/xeokit/xeokit-server.d.ts

```diff
@@ -1 +1,7 @@
-export class XeokitServer {}
\ No newline at end of file
+import {GonType} from "../../../common/gon/gon.service";
+
+declare global {
```

Comment:
Because this interface is already defined in the GonService as global (declare global), it seems that it shouldn't be needed here.